app/src/main/java/skills/future/planer/db/habit/HabitDuration.java

```diff
@@ -0,0 +1,17 @@
+package skills.future.planer.db.habit;
+
+public enum HabitDuration {
+    UltraShort(21),
+    Short(90),
+    Long(120);
```

Comment:
Nie można było jakoś lepiej tego nazwać? XD

Reply:
jak masz lepszy pomysł to droga wolna, tylko zmień te nazwy w całym projekcie